backend/packages/Upgrade/src/api/services/SegmentService.ts

```diff
@@ -616,4 +624,8 @@ export class SegmentService {
       .getRepository(Segment)
       .findOne({ where: { id: segmentDoc.id }, relations: ['individualForSegment', 'groupForSegment', 'subSegments'] });
   }
+
+  private trimAndRemoveHiddenChars(value: string): string {
+    return value.trim().replace(/[\r\n]/g, '');
```

Comment:
It's probably even less likely, but it might be good to include '\t' in the trimming too.